util/pytest_log_filter.py

```diff
@@ -0,0 +1,16 @@
+import re
+import sys
+
+
+def filter_test_log(log):
+    with open(log) as f:
+        log_txt = f.read()
+    test_results = re.search("\n\n===(.+?)((.|\n)*)==", log_txt).group()  # Filter to only get the usefull parts of the info
```

Comment:
It's a good idea to put in a comment an example of input that the regex is supposed to catch. (Otherwise here, it's defined only as "usefull part", which is not very precise.